go.mod

```diff
@@ -2,14 +2,12 @@ module github.com/unweave/cli
 
 go 1.19
 
-replace (
-	github.com/unweave/unweave => ../infra/unweave
-)
+replace github.com/unweave/unweave => ../infra/unweave
 
 require (
 	github.com/charmbracelet/lipgloss v0.6.0
+	github.com/davecgh/go-spew v1.1.1
```

Comment:
Where is this used 🤔

Reply:
ah need to undo this, spew is a debugging tool